### DISCUSSION TITLE
Terralith Datapack

### DISCUSSION THREAD

**Discussion by ghost** [EnigmaticaModpacks/Enigmatica8#2] [Ideas]
This would be a great terrain/biome gen for Enigmatica 8

https://www.youtube.com/watch?v=OszX5u97Sik

https://www.curseforge.com/minecraft/mc-mods/terralith

Really impressed with this datapack and in my playing does not cause alot of lag.

Just an idea, delete this if you want to. :)

**Comment by NielsPilgaard** [comment]
Agreed that would be awesome, but I'd prefer if it was a separate dimension so players can still see the new vanilla worldgen. 😀

**Reply by ghost** [reply]
That would be even more awesome.

**Comment by shaksternano** [comment]
Unfortunately I don't think terralith is compatible with mods that do worldgen.

**Reply by ghost** [reply]
With what Terralith can do with worldgen E8 wouldnt need anything else.  It was just an idea.

**Comment by TemperedFool** [comment]
I'd kinda like to see what vanilla worldgen + structures and mobs to fill the world up would look like before we resort to anything drastic...

**Comment by theboo** [comment]
So I threw terralith on my personal server... NGL it really doesn't change the 1.18 feel at all(I spent a couple hours flying through with and without terralith in spectator mode), terrain seems largely the same... Adds some cool vanilla block based biomes and tweaks the vanilla biomes a bit .

The one downside I can see is I don't think it will work with other biome mods

**Reply by xATL4S** [reply]
I was watching this video and while this is another mod but for the nether - its by the same person so maybe there is some chance of it working with other biome mods, He mentions that the nether mod works with "Better nether" while still giving the increased world height and all.

https://www.youtube.com/watch?v=S3y93zRGHwI&ab_channel=AsianHalfSquat - link for vieo if interested (I don't really know how to add time stamps to url's but it mentions the compat with the mod "Better Nether" at about 2:13) (mod that increases word height for the nether and includes 3D biomes)

**Comment by theboo** [comment]
![Screenshot_20211212-075506](https://user-images.githubusercontent.com/3585761/145719455-beb69ebb-d2f7-4723-97ec-cc905aabb439.png)

**Reply by ghost** [reply]
This was fixed in v2.0.5
